.github/workflows/web-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Web CI
+
+on:
+  workflow_dispatch:
```

Comment:
maybe we can also do automatic release?

Reply:
The issue with automatic release is the version number, we should have something to check if the update is minor or major such as semantic release plugin that checks commits.

Reply:
How about this one? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#create
Will be triggered every time we create a new github release (only manually AFAIK)

Reply:
Found a better solution: `push:
    tags:
      - "v[0-9]+.[0-9]+.[0-9]+"`
Needs testing 😶‍🌫️

Reply:
Let's test it then 👀